src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+
+  return (men.reduce((sumOfAges, man) =>
+    sumOfAges + (man.died - man.born), 0)) / men.length;
```

Comment:
```suggestion
  return men.reduce((sumOfAges, man) => (
      sumOfAges + (man.died - man.born)
    ),
    0,
  ) / men.length;
```

Reply:
Fix here and in the whole file

More details here
https://mate-academy.github.io/style-guides/javascript.html#arrow-functions